minio/helpers.py

```diff
@@ -479,7 +479,7 @@ def is_valid_bucket_notification_config(notifications):
                     'configuration'
                 )
             # check if 'Id' key is present, it should be a string
```

Comment:
I think 'string' in this comment now includes both bytes and str (as imported from .compat). Could you update the comment to reflect it.